### ISSUE TITLE
False negative - override detection

### ISSUE THREAD

**Issue by Dragon-32** [X2CommunityCore/xcom2-launcher#148]
[[WOTC] Facility Clearing UI Indicator Fix](https://steamcommunity.com/sharedfiles/filedetails/?id=1130714554) overrides UIFacilityGrid_FacilityOverlay, as does [[WotC] RW Realistic Non Self Building Facilities](https://steamcommunity.com/sharedfiles/filedetails/?id=1151690543) however the Community Mod Launcher isn't detecting these overrides as conflicts.

See Lebowskichild's comments from 29th September in the [[WOTC] Facility Clearing UI Indicator Fix](https://steamcommunity.com/sharedfiles/filedetails/?id=1130714554) Comments. They've been suffering typical mod conflict symptoms but without the Community Mod Launcher highlighting the conflict.

**Comment by robojumper** [comment]
[WOTC] Facility Clearing UI Indicator Fix
```
[Engine.Engine]
+ModClassOverrides=(BaseGameClass="UIFacilityGrid_FacilityOverlay", ModClass="UIFacilityGrid_fixed")
```

[WotC] RW Realistic Non Self Building Facilities
```
[Engine.Engine]
+ModClassOverrides=(BaseGameClass="UIFacilityGrid_FacilityOverlay",					  ModClass="Override_UIFacilityGrid_FacilityOverlay"				)
```

It looks like the parsing code fails to strip tab characters:

https://github.com/X2CommunityCore/xcom2-launcher/blob/9623e9414ba9977b4501c4ae89a8b042ab251dd7/xcom2-launcher/xcom2-launcher/Classes/Mod/ModEntry.cs#L236-L242

In fact, none of [WotC] RW Realistic Non Self Building Facilities' overrides are being detected. Marking as Bug and Accepted.

**Comment by Xymanek** [comment]
Don't we have a proper ini parser? Why is this using a regex?

**Comment by RevZero** [comment]
I never found the time to take a look at the mod conflict detection stuff. Maybe we can fix/improve this in v1.4.

Related to #102.

**Comment by RevZero** [comment]
Resolved in 144bffff6448bbe12275eab296d189de7fb68998.
